MANIFEST.in

```diff
@@ -1,8 +1,5 @@
 recursive-include requirements *
-recursive-include signingscript/test *
-recursive-include signingscript/data *
-
-recursive-include signingscript/vendored *
+recursive-include src/signingscript
```

Comment:
If you run `python setup.py sdist` and then `tar tzvf dist/signingscript-12.1.0.tar.gz`, you'll note that we lose `data/` and `vendored/` from the source tarball. We need the trailing `*` here.